natural-language-understanding/src/main/java/com/ibm/watson/natural_language_understanding/v1/model/CategoriesOptions.java

```diff
@@ -24,13 +24,13 @@
 
   protected Boolean explanation;
   protected Long limit;
-  @Deprecated protected String model;
+  protected String model;
```

Comment:
I think this is deprecated

Reply:
Should I just add @Deprecated back to it?

Reply:
Yes